apps/prairielearn/src/migrations/20240415170055_institution_administrators__create.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS institution_administrators (
```

Comment:
We are calling this table `institution_administrators` but the pages that view it are called `institutionAdminAdmins`. I'm already starting to get confused about when I should use `administrator` vs `admin`. Should we standardize on one or the other?

I guess this was originally my fault for not calling the `adminstrators` table `admins` 😭

Reply:
This page name was chosen to match two existing conventions:

- The adjacent pages for managing institution stuff are all `institutionAdmin*`
- The similar page for managing global administrators is called `administratorAdmins`

Combining those, I got `institutionAdminAdmins`. I'm not saying I love it, but that's the rationale. If we're going to change either of the things I'm matching against, let's do that in another PR?

Reply:
Works for me!